paddle/framework/program_desc.h

```diff
@@ -31,8 +31,6 @@ class ProgramDescBind {
 
   BlockDescBind *Block(size_t idx) { return blocks_[idx].get(); }
 
-  std::string DebugString() { return Proto()->DebugString(); }
```

Comment:
`Protobuf::DebugString` is using the reflection of Protobuf. It is not available for `PROTO_LITE`.

Reply:
So we can not use `DebugString` anymore?

Reply:
No, we cannot.